contracts/interfaces/ILegacyDCAHub.sol

```diff
@@ -36,14 +38,6 @@ interface ILegacyDCAHub {
     bytes1 intervalsInSwap;
   }
 
-  /// @notice A pair of tokens, represented by their indexes in an array
-  struct PairIndexes {
```

Comment:
I removed this struct here and I'm using the one from the new DCAHub because Solidity can't tell that they are actually the same. So in order to be able to create a pair that can be used for both the legacy and new hubs, they need to be the same

Reply:
I was wondering why that change. Good to know.